src/main/resources/leonardo-example.conf

```diff
@@ -0,0 +1,113 @@
+# Example conifg file for leonardo.
+# Anything commented out here will have have defaults specified or is not necessary to run leo.
+# All values can be modified for your specific google environment, database, and secruity certificates
+
+
+# Google Cloud dataproc configuration
+dataproc {
+  applicationName = "leonardo"
+  dataprocDefaultRegion = "us-central1"
+  leoGoogleProject = ""
+  clusterUrlBase = "https://leonardo.dsde-dev.broadinstitute.org/notebooks/"
```

Comment:
Should we be specifying our real dev environment in the example config?

Reply:
nope